### PR TITLE
docs(repositories): add themes section

### PR DESCRIPTION
**Pull request by danilowoz**
Closes PC-1080

### CODE REVIEW

File: packages/projects-docs/pages/learn/repositories/themes.mdx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: Themes
+description: Change the color of your editor by utilizing the preset themes available in VS Code.
```
Comment (alexnm) [inline packages/projects-docs/pages/learn/repositories/themes.mdx:3]: nitpick: I would replace utilizing with using but would also go for a different description

```
Choose a different color scheme for your editor by selecting one of the preset themes.
``` 

I find the VS Code reference here a bit confusing, the themes are standalone color styles, not really related to VS Code in my mind.

File: packages/projects-docs/pages/learn/repositories/themes.mdx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+# Themes
+
+Change the color of your editor by utilizing the preset themes available in VS Code.
+
+![CodeSandbox Themes](../images/themes.jpg)
+
+## Overview
+
+CodeSandbox maps themes of Visual Studio Code extensions to apply over the editor. Currently, it only offers a set of preset themes, including various dark and light themes:
```
Comment (alexnm) [inline packages/projects-docs/pages/learn/repositories/themes.mdx:16]: Same as above

```CodeSandbox maps popular themes from the dev community to apply over the editor.```

File: packages/projects-docs/pages/learn/repositories/themes.mdx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+- [GitHub's Default (dark / light)](https://marketplace.visualstudio.com/items?itemName=GitHub.github-vscode-theme)
+- [Vesper (dark)](https://marketplace.visualstudio.com/items?itemName=raunofreiberg.vesper)
+- [Dracula (dark)](https://marketplace.visualstudio.com/items?itemName=dracula-theme.theme-dracula)
+- [poimandres (dark)](https://marketplace.visualstudio.com/items?itemName=pmndrs.pmndrs)
+
+## Configure
+
+There are two ways to switch themes:
+- Access the command palette in the editor using <kbd>⌘</kbd> + <kbd>K</kbd> and search by `themes`.
+- Open the Project settings and select the **Themes** tab.
```
Comment (alexnm) [inline packages/projects-docs/pages/learn/repositories/themes.mdx:28]: I would move this one first, usually the command palette is less used and probably people are looking for some visual setting somewhere in the menu.

File: packages/projects-docs/pages/learn/repositories/themes.mdx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+CodeSandbox maps themes of Visual Studio Code extensions to apply over the editor. Currently, it only offers a set of preset themes, including various dark and light themes:
+
+- CodeSandbox (dark / light)
+- [GitHub's Default (dark / light)](https://marketplace.visualstudio.com/items?itemName=GitHub.github-vscode-theme)
+- [Vesper (dark)](https://marketplace.visualstudio.com/items?itemName=raunofreiberg.vesper)
+- [Dracula (dark)](https://marketplace.visualstudio.com/items?itemName=dracula-theme.theme-dracula)
+- [poimandres (dark)](https://marketplace.visualstudio.com/items?itemName=pmndrs.pmndrs)
+
+## Configure
```
Comment (alexnm) [inline packages/projects-docs/pages/learn/repositories/themes.mdx:24]: Maybe a more specific headline, `Change theme`? might be better for SEO / Accessibility

**Review by alexnm — CHANGES_REQUESTED** [review]
Just some suggestions, not blocking

File: packages/projects-docs/pages/learn/repositories/themes.mdx (hunk 2 above)
Comment (filipelima18) [reply packages/projects-docs/pages/learn/repositories/themes.mdx]: Proposing an updated version below that might be easier to read:

"CodeSandbox provides some popular themes created by the dev community that you can apply to the cloud editor. Currently, we only offer a few preset themes, with dark and light options:"

File: packages/projects-docs/pages/learn/repositories/themes.mdx [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+CodeSandbox maps popular themes from the dev community to apply over the editor. Currently, it only offers a set of preset themes, including various dark and light themes:
+
+- CodeSandbox (dark / light)
+- [GitHub's Default (dark / light)](https://marketplace.visualstudio.com/items?itemName=GitHub.github-vscode-theme)
+- [Vesper (dark)](https://marketplace.visualstudio.com/items?itemName=raunofreiberg.vesper)
+- [Dracula (dark)](https://marketplace.visualstudio.com/items?itemName=dracula-theme.theme-dracula)
+- [poimandres (dark)](https://marketplace.visualstudio.com/items?itemName=pmndrs.pmndrs)
+
+## Change theme
```
Comment (filipelima18) [inline packages/projects-docs/pages/learn/repositories/themes.mdx:24]: "Changing the theme" - to keep consistency with the other docs pages

**Pull request merged**